### PR TITLE
Update CONTRIBUTING.md Grammar fixure.

### PR DESCRIPTION
**Pull request by CommaNet**
Added a comma, which was needed.

### CODE REVIEW

**Comment by DHowett** [comment]
Sorry for the delay in reviewing this! Much of the team is unavailable this week, so we're short-staffed. We'll get to the backlog of issues and PRs as soon as possible.

**Review by carlos-zamora — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -162,4 +162,4 @@ Once your code has been reviewed and approved by the requisite number of team me
 
 ## Thank you
 
-Thank you in advance for your contribution! Now, [what's next on the list](https://github.com/microsoft/terminal/labels/Help%20Wanted)? 😜
+Thank you in advance for your contribution! Don't be sad if your contribution/pull request was not accepted because of a certain issue. You will still be able to contribute in the future. We still appreciate you taking time to ensure the best for this project! Now, [what's next on the list](https://github.com/microsoft/terminal/labels/Help%20Wanted)? 😜
```
[the remark this answers is not in the document]
Comment (CommaNet) [reply CONTRIBUTING.md:165]: That shall do it then.

**Comment by CommaNet** [comment]
Closed due to inactivity.

**Comment by lhecker** [comment]
@CommaNet I was under the impression that "That shall do it then." means you're going to change it.
Please feel free to reopen the PR at any time if you'd like to change it. 🙂

**Pull request closed without merging**